### ISSUE TITLE
Not able to render the docs due to faulty metadata file?

### ISSUE THREAD

**Issue by aadsm** [facebook/Docusaurus#342]
```
Error: Improper sidebars.json file, document with id 'advanced-topics-inline-styles' not found. Make sure that documents with the ids specified in sidebars.json exist and that no ids are repeated.
    at _loop (/Users/aadsm/Projects/draft-js/website/node_modules/docusaurus/lib/server/readCategories.js:58:17)
```
I've debug the code in `readCategories.js` and noticed it's cycling the `metadata.js` that is automatically generated. I believe there's a problem with how this file handles translated and non-translated ids as it mixes both in it:

```
"en-advanced-topics-block-components": {
    "id": "en-advanced-topics-block-components",
    "title": "Custom Block Components",
    "layout": "docs",
    "category": "Advanced Topics",
    "next": "advanced-topics-inline-styles",
    "permalink": "docs/advanced-topics-block-components.html",
    "source": "Advanced-Topics-Block-Components.md",
    "localized_id": "advanced-topics-block-components",
    "language": "en",
    "next_title": "Next"
  },
```

The code in question: `articles[metadata.next]`, checks if there is an entry in metadata for the `next` property, however, the `next`is a non-localized id, while the metadata entries use localized ids, because of this it never finds the next, leading to error.

**Comment by JoelMarcey** [comment]
@aadsm Hi. I assume you are running Docusaurus 1.0.3 via `docusaurus-init`. This may be fixed in `master` that will go out with 1.0.4, which I hope to get out sometime this week. 

To check to see if you get the error still, we could try installing the latest Docusaurus from `master` for `draft-js` and testing via the instructions here: https://github.com/facebook/Docusaurus/blob/master/admin/local-third-party-project-testing.md#install-from-a-local-docusaurus-repo

Or I can test this for you if you have a repo available that has your website.

**Comment by aadsm** [comment]
I've updated to master using the instructions but still have the same error :(.
Is there a way to completely avoid using languages? I'd imagine most projects only have one language also.

**Comment by JoelMarcey** [comment]
@aadsm We are updating the code to make it so that the default is not to have a set language for the links and ids unless a `lanaguges.js` file is present. 

Is there anyway you can give me access to the repo where you are developing the site so I can run some tests too?

**Comment by JoelMarcey** [comment]
Hi @aadsm - can you let me know if this issue is resolved in Docusaurus 1.0.7?

**Comment by endiliey** [comment]
Hi @aadsm. I'm closing this issue due to inactivity & there seems to be no issue related to this now. Feel free to reopen if you need assistance 😄
